Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
```

Comment:
This only matters when you are running the specs, otherwise if some dependency is upgraded then the tests might start to break.